src/components/FilterItem/FilterItem.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import createClass from 'classnames';
+
+const FilterItem = ({
+  href,
+  dataFilter,
+  onClick,
+  anchor,
+  filter
+}) => {
+  const linkOfFilterClass = createClass({'selected': dataFilter === filter});
```

Comment:
place the object on the new line for better readability

Reply:
Fixed!